src/scitokens_xrootd.py

```diff
@@ -91,7 +94,17 @@ def config(fname):
         if ie.errno == errno.ENOENT:
             return
         raise
+        
     for section in cp.sections():
+        if section.lower() == 'global':
+            if 'audience_json' in cp.options(section):
+                # Read in the audience as json.  Hopefully it's in list format or a string
+                g_audience = json.loads(cp.get("Global", "audience_json"))
+            elif 'audience' in cp.options(section):
+                g_audience = cp.get("Global", "audience")
+                if ',' in g_audience:
```

Comment:
Can you always split the `g_global_audience`?  This way, `g_audience` is always a list and never a string.

Reply:
Sure, I can always make `g_audience` a list, though the underlying library pyjwt doesn't really care.